### PR TITLE
Update design for storing input channels in HPA dataset

### PR DESCRIPTION
**Pull request by anwai98**
Takes care of https://github.com/constantinpape/torch-em/issues/350.

I made a minor addition to the way we treat multiple groups (i.e. `raw_key` / `label_key`) passed by the user and then likewise stack inputs from multiple groups. Let me know if you think there is a better way to handle this.

### CODE REVIEW

File: torch_em/data/datasets/light_microscopy/hpa.py [hunk 1]
```diff
@@ -370,7 +370,7 @@ def get_hpa_segmentation_dataset(
     Returns:
        The segmentation dataset.
     """
-    get_hpa_segmentation_data(path, download, channels, n_workers_preproc)
+    get_hpa_segmentation_data(path, download, n_workers_preproc)
```
Comment (constantinpape) [inline torch_em/data/datasets/light_microscopy/hpa.py:373]: We should add a check here to ensure that `channels` passed by the user are valid.

File: scripts/datasets/light_microscopy/check_hpa.py [hunk 2]
```diff
@@ -10,7 +10,14 @@
 def check_hpa():
     from util import ROOT
 
-    loader = get_hpa_segmentation_loader(os.path.join(ROOT, "hpa"), "train", (512, 512), 1, download=True)
+    loader = get_hpa_segmentation_loader(
+        path=os.path.join(ROOT, "hpa"),
+        split="train",
+        patch_shape=(1024, 1024),
+        batch_size=1,
+        channels=["protein"],
```
Comment (constantinpape) [inline scripts/datasets/light_microscopy/check_hpa.py:18]: Have you checked that this works when using multiple channels?

**Review by constantinpape — COMMENTED** [review]
This is a very nice solution! Only two minor comments.

File: scripts/datasets/light_microscopy/check_hpa.py (hunk 2 above)
Comment (anwai98) [reply scripts/datasets/light_microscopy/check_hpa.py]: Yes, it works as expected!

**Review by constantinpape — APPROVED** [review]

**Pull request merged**